bin/janus

```diff
@@ -0,0 +1 @@
+run-command
```

Comment:
Check it!  I make ./bin/janus and ./bin/magma etc files that can be used outside of `make -C metis|janus|magma bash`.  It's just a convenient wrapper.

Reply:
Hmm, for me there is always a considerable amount of overhead (~10s) + output when I run make bash, so I usually just keep it open while running specs or whatnot. Is this avoidable?

Reply:
I feel like before we changed to a shared image, it was super fast to bring up a bash terminal because it didn't re-run the `build.sh` script every time?

Reply:
You can still keep the persistent console open, nothing to prevent that.  This interface is just "in addition".  It's most useful for when I want to quickly run a janus one-off but I don't need that persistent console.

Yeah, I could probably do some work to reduce the amount of rebuilding the system does.  The shared image work reduced building in some cases, but the etna gem rebuild is triggered aggressively, I think, due to it not filtering files when it checks for changes.  I can sneak that change in soon.

Reply:
Actually, I now see what may be causing the excessive rebuilds.  It's tracking every node_modules change as needing a rebuild of the base image, but because that directory is mounted, every restarted updates it anyways.  I can filter that in this PR to make that issue go away.